client-app/pages/product.vue

```diff
@@ -216,22 +215,24 @@ const seoDescription = computed(() => product.value?.seoInfo?.metaDescription);
 const seoKeywords = computed(() => product.value?.seoInfo?.metaKeywords);
 const seoImageUrl = computed(() => product.value?.imgSrc);
 
-const productInfoSection = computed(() =>
-  template.value?.content.find((item: PageContent) => item?.type === "product-info"),
-);
+const productTemplate = computed(() => _productTemplate as PageTemplate);
```

Comment:
is creating a computed property required here? seems that there is no reactivity and it can be substitute with const